packages/ui-core/core/src/lib/services/favorite/favorite-store.service.ts

```
@@ -0,0 +1,126 @@
+					});
+				});
+			favoritePromises.push(promise);
+		}
+
+		const allFavoriteItems = await Promise.all(favoritePromises);
+		return allFavoriteItems.flat();
+	}
+
+	private _truncateTitle(title: string, maxLength = 24): string {
+		if (!title) return '';
+		return title.length > maxLength ? title.slice(0, maxLength - 3) + '...' : title;
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)